src/events/messageReactionRemove.ts

```diff
@@ -51,10 +61,20 @@ export default async (client: Discord.Client, reaction: Discord.MessageReaction,
         return await safeSendMessage(channel, '⚠️ Your vote was not possible to remove. (Internal error)')
       }
 
-      // Don't need to check for approval or rejection here as submissions are deleted upon approval/rejection
-      // and a removal of a vote can never push a project beyond a positive threshold
+      log.info(`User ${user.id} (${user.tag}) removed their ${isUpvote ? 'upvote' : isPause ? 'suspend' : 'downvote'} for project ${project.name} (ID ${project.id})`)
+      // Since we are now pausing the submission, the rejection upon pause is possible. We will check submission here.
+      const input: ShowcaseInput = {
+        result,
+        isPause
+      }
 
-      log.info(`User ${user.id} (${user.tag}) removed their ${isUpvote ? 'upvote' : 'downvote'} for project ${project.name} (ID ${project.id})`)
+      const dInput: ShowcaseDiscordInput = {
+        guild,
+        channel,
+        user,
+        reaction
+      }
+      await showcase(dInput, input)
```

Comment:
Descriptive variable names, please.

See also: My previous about uncaught error from the `showcase` function
```suggestion
      const discordInput: ShowcaseDiscordInput = {
        guild,
        channel,
        user,
        reaction
      }
      await showcase(discordInput, input)
```